config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -30,26 +30,31 @@ presubmits:
           requests:
             cpu: 4
 
-  - name: pull-kubernetes-verify-prow
+  # TODO(krzyzacy): add branches once this works
+  # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
+  - name: pull-kubernetes-verify-podutil
```

Comment:
happy to add this, but IIRC it depends on golint being in the image so

Reply:
it's currently using kubekins-test which has golint :-)